### DISCUSSION TITLE
video_tags metadata in MP4 for Plex

### DISCUSSION THREAD

**Discussion by aaronmitchellharris** [jmbannon/ytdl-sub#1126] [Q&A]
Does anyone know the correct `video_tags` values to use when embedding metadata in MP4 video files to be used in Plex?

So far, I've found that these `video_tags` correspond to these Plex values:
`date` -> `Originally Available`
`genre` -> `Genres`
`album` -> `Collection`

However, I'm having a hard time figuring out which `video_tags` will work for things like `Studio`, `Directors`, `Writers`, `Producers`. I'd like to use these tags as ways to organize my videos.

I used https://github.com/squaresmile/Plex-Plug-Ins/blob/master/LocalMedia.bundle/Contents/Code/videohelpers.py to figure out I needed to use `album` for `Collection` since that one was less obvious, but the other things I've tried from there don't seem to work.

**Comment by jmbannon** [comment]
I'm not familiar, the Plex forums are probably best to ask this question. Plz report back here if you do find out!
